clickstream/docs/index.rst

```diff
@@ -36,6 +36,15 @@ are quite large and depending on your network connection may take
        cd examples/clickstream
        git checkout |release_post_branch|
 
+#. Get the Jar files for the source and sink connectors
+
+   .. code:: bash
+
+       PWD=$(pwd)
```

Comment:
@bbejeck this may not be required.  Based on observation in my environment, `PWD` is always defined.

Reply:
ack, updated